dashboard/db/schema.rb

```diff
@@ -39,6 +39,24 @@
     t.index ["lesson_activity_id"], name: "index_activity_sections_on_lesson_activity_id"
   end
 
+  create_table "ai_tutor_interactions", charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+    t.integer "user_id", null: false
+    t.integer "level_id"
+    t.integer "script_id"
+    t.string "ai_model_version"
+    t.string "type"
+    t.string "project_id"
+    t.string "project_version_id"
+    t.text "prompt"
+    t.string "status"
+    t.text "ai_response"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
```

Comment:
I think we also want a `deleted_at` column. Does that sound right?

Reply:
I think there was a comment from the meeting (from @sureshc maybe?) that we'd want all the "ActiveRecord" columns, including created/updated/deleted, with the idea that we'd probably use the deleted_at column if we delete all our data (or soft-delete/anonymize) 30 days after the pilot (or whatever the requirement is). If we don't need this column until then, that's fine, too!

Reply:
@ebeastlake the scenario I'm aware of where you will need this is when you go to implement soft-deletion of ai tutor chat history when a user deletes their account. IIRC the plan you and Erin shared was to NOT implement this PII cleanup for now, and instead to delete all pilot chat data after the pilot ends. If this is correct, then is there another reason you want to be able to delete chat data (there are certainly valid scenarios here, e.g. if a teacher reports something and you want to hide it from them but not eliminate it from our system?)

Reply:
Ah, thanks for this question, @davidsbailey! It highlights the point of confusion. When I suggested we add this column, I thought, "Is there any universe in which our 'hard deletion' of all pilot chat data after the pilot ends pivots to a 'soft delete' with the help of this column (and Travis's approval, of course) in a way that allows us to retain valuable data while still complying with the requirement?" But maybe if that seems like an option when we get there, we can add it then.

Reply:
@ebeastlake apologies, it looks like I may have missed your previous comment when I posted my previous comment. if you have plans for the deleted_at column then it seems fine to add it now. it is a judgment call. if no one feels strongly then I am happy for you to decide.

Reply:
I left `deleted_at` off for now because my understanding, as Darin and Dave alluded to, was that we would completely delete everything from the pilot without needing a record of when it was deleted. Then, when we have more clarity about whether full production data is going to be hard-deleted, soft-deleted, or anonymized/obfuscated in some way we could add a column that would indicate more precisely what's happening.  Does that sound right?

Reply:
If we add it, it'll be null for every existing row, so it shouldn't be a costly operation.